### DISCUSSION TITLE
New Ride Type: Mega Coaster

### DISCUSSION THREAD

**Discussion by karst** [OpenRCT2/OpenRCT2#21014] [Feature requests]
Based on Intamin's new track type, such as Skyrush, Red Force, Kondaa, VelociCoaster, Hyperion, Toutatis, etc.

This will use the same spine mechanics as is currently in the works for the giga coaster track (Soon LSM/Multi-Looping Coaster track too)

It won't be much different in game mechanics from the twister coaster, outside of looks. Perhaps slight different balancing.

**Comment by Realsteel89** [comment]
Can we add the more modern track that is this track in real life using because we can do this now like velocity coaster has or phantom I mean the double spine track as new track peace for this

**Reply by karst** [reply]
That's the idea. Though I believe it's been decided to wait on adding more track types until track type objects are a thing.

**Reply by Realsteel89** [reply]
but that's a pity because we have the possibility to generate them more or less quickly and good
